src/utils/signOut.tsx

```diff
@@ -0,0 +1,11 @@
+import { NavigateFunction } from 'react-router-dom';
+import { AuthContextType } from '../components/Auth/Auth';
+
+function signOutHandler(auth: AuthContextType, navigate: NavigateFunction) {
+  return () => {
+    localStorage.removeItem(`${import.meta.env.VITE_PRODUCT_NAME}_user`);
```

Comment:
Since with Session authentication we don't save anything in the local storage, this is not needed. 

On the other hand, we will need an API call to logout the user.

Reply:
I will refactor the complete sign in/out after the backend is running local.